pkg/api/types.go

```diff
@@ -597,10 +597,34 @@ const (
 	ClaimLost PersistentVolumeClaimPhase = "Lost"
 )
 
+type HostPathType string
+
+const (
+	// For backwards compatible, leave it empty if unset
+	HostPathUnset HostPathType = ""
```

Comment:
This is for backwards compatible. If a volume that has this value, it will behave like before, doing nothing.